### DISCUSSION TITLE
How to use a trait in a Stat Block

### DISCUSSION THREAD

**Discussion by FilBot3** [Obsidian-TTRPG-Community/fantasy-statblocks#104] [Q&A] [answered]
Hello!

I am attempting to create a custom Layout for the Savage World Adventure Edition system and I wanted to use the `trait` block because from what I can glean from the README, it looks like a List of Objects that have the following schema:

```yaml
traits:
  - name: ''
    desc: ''
```

However, when I render this stat block, nothing shows up except the header. I don't think I'm using this block properly, but the documentation doesn't really elaborate on how to use it in a custom layout.

**Comment by FilBot3** [comment, marked as answer]
It seems that the `trait` block wouldn't render in an `Inline Group` or a `Group` with multiple Blocks in it. So I had to move them to their own Groups with a single Trait Block in it.

**Reply by kylomax** [reply]
This is actually only an issue with inline groups. You can nest as many groups as you want, but inline groups can't be nested for some reason. When inline groups are nested the content in the nested inline group doesn't show at all. There's actually no issue opened for this bug, but I am opening one right now.

**Reply by FilBot3** [reply]
I think I saw that traits block didn't render properly if I had 2 or more in the same Group block.
